pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[tool.poetry]
+name = "hexlet-code"
+version = "0.1.0"
+description = "simple tool for web-page analyzing"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+authors = ["alexandra <momotav717@gmail.com>"]
+readme = "README.md"
+packages = [
+    {include = "page_analyzer"},
+]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+python-dotenv = "^1.0.0"
+
+[tool.poetry.group.dev.dependencies]
+flask = "^3.0.0"
+gunicorn = "^21.2.0"
```

Comment:
flask and gunicorn are main project dependencies, they shouldn't be in dev group